### DISCUSSION TITLE
build doris-manager faild

### DISCUSSION THREAD

**Discussion by mchades** [apache/doris#6886] [A - General / Q&A] [answered]
I run `npm install` in doris-manager on branch doris-manager, but got an error below.Where can I found @baidu/aidp-middle-common@^1.3.18 ?
![图片](https://user-images.githubusercontent.com/24897598/137900652-00281152-1832-4bec-b74a-b9537b5d5b02.png)

**Comment by morningman** [comment, marked as answer]
The doris-manage branch is under developed. Not ready for use.
